ffmpeg/videoprofile.go

```diff
@@ -73,6 +93,12 @@ var (
 	P240p30fps4x3  = VideoProfile{Name: "P240p30fps4x3", Bitrate: "600k", Framerate: 30, AspectRatio: "4:3", Resolution: "320x240"}
 	P144p30fps16x9 = VideoProfile{Name: "P144p30fps16x9", Bitrate: "400k", Framerate: 30, AspectRatio: "16:9", Resolution: "256x144"}
 	P144p25fps16x9 = VideoProfile{Name: "P144p25fps16x9", Bitrate: "400k", Framerate: 25, AspectRatio: "16:9", Resolution: "256x144"}
+	PDnnDetector   = VideoProfile{Name: "PDnnDetector", Bitrate: "400k", Framerate: 20, AspectRatio: "1:1", Resolution: "224x224",
+		Detector: DetectorProfile{SampleRate: 30, ModelPath: "tasmodel.pb", Threshold: 0.8, Input: "input_1", Output: "reshape_3/Reshape",
+			ClassIds: []int{1}, ClassNames: []string{"adult", "soccer"}}}
+	PDnnVioFilter = VideoProfile{Name: "PDnnVioFilter", Bitrate: "400k", Framerate: 20, AspectRatio: "1:1", Resolution: "224x224",
+		Detector: DetectorProfile{SampleRate: 30, ModelPath: "tviomodel.pb", Threshold: 0.8, Input: "input_1", Output: "reshape_3/Reshape",
+			ClassIds: []int{0}, ClassNames: []string{"violence"}}}
```

Comment:
Given that we neither use the output frames or perform any FPS/Scaling filtering when configured with these DNN profiles, does it make sense to remove `Bitrate`, `Framerate`, `Resolution` and other redundant options from here to avoid confusion for the reader?

Reply:
I agree, removed Bitrate, Framerate, Resolution, AspectRatio.